[2017-07-07 15:38:57,815][INFO ][main]
[2017-07-07 15:38:57,828][INFO ][main]Could not initialize class com.tuhanbao.thirdapi.cache.redis.RedisConfig
[2017-07-07 15:38:57,829][WARN ][main]no config file for : ftp
[2017-07-07 15:38:57,842][INFO ][main]
[2017-07-07 15:49:34,464][WARN ][main]no config file for : redis
[2017-07-07 15:49:34,516][WARN ][main]no config file for : ftp
[2017-07-07 15:49:34,529][INFO ][main]
[2017-07-07 15:50:35,285][WARN ][main]no config file for : redis
[2017-07-07 15:50:35,343][WARN ][main]no config file for : ftp
[2017-07-07 15:50:35,357][INFO ][main]
[2017-07-07 15:52:49,800][WARN ][main]no config file for : redis
[2017-07-07 15:52:49,851][WARN ][main]no config file for : ftp
[2017-07-07 15:52:49,863][INFO ][main]
[2017-07-07 16:29:20,034][WARN ][main]no config file for : redis
[2017-07-07 16:29:20,088][WARN ][main]no config file for : ftp
[2017-07-07 16:29:20,101][INFO ][main]
[2017-07-07 16:29:22,406][INFO ][main]java.io.FileInputStream.open(Native Method)
[2017-07-07 16:29:22,406][INFO ][main]java.io.FileInputStream.<init>(FileInputStream.java:146)
[2017-07-07 16:29:22,406][INFO ][main]java.io.FileInputStream.<init>(FileInputStream.java:101)
[2017-07-07 16:29:22,406][INFO ][main]com.tuhanbao.base.util.io.excel.ExcelUtil.read(ExcelUtil.java:211)
[2017-07-07 16:29:22,406][INFO ][main]com.tuhanbao.autotool.mvc.excel.ParseExcelFilter.readExcel(ParseExcelFilter.java:40)
[2017-07-07 16:29:22,406][INFO ][main]com.tuhanbao.autotool.mvc.excel.ParseExcelFilter.readFile(ParseExcelFilter.java:52)
[2017-07-07 16:29:22,406][INFO ][main]com.tuhanbao.autotool.mvc.excel.ParseExcelFilter.readFile(ParseExcelFilter.java:48)
[2017-07-07 16:29:22,406][INFO ][main]com.tuhanbao.autotool.mvc.excel.ParseExcelFilter.initProjectByConfig(ParseExcelFilter.java:30)
[2017-07-07 16:29:22,406][INFO ][main]com.tuhanbao.autotool.mvc.excel.ParseExcelFilter.filter(ParseExcelFilter.java:23)
[2017-07-07 16:29:22,406][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-07-07 16:29:22,406][INFO ][main]Test.main(Test.java:18)
[2017-07-07 16:29:22,406][ERROR][main]java.io.FileNotFoundException: E:\git\tuhanbao\demo\config\~$config_debug.xlsx (另一个程序正在使用此文件，进程无法访问。)
[2017-07-07 16:29:31,284][WARN ][main]no config file for : redis
[2017-07-07 16:29:31,335][WARN ][main]no config file for : ftp
[2017-07-07 16:29:31,348][INFO ][main]
[2017-07-07 16:30:58,945][WARN ][main]no config file for : redis
[2017-07-07 16:30:58,998][WARN ][main]no config file for : ftp
[2017-07-07 16:30:59,011][INFO ][main]
[2017-07-07 16:31:42,198][WARN ][main]no config file for : redis
[2017-07-07 16:31:42,256][WARN ][main]no config file for : ftp
[2017-07-07 16:31:42,269][INFO ][main]
[2017-07-07 16:31:54,813][WARN ][main]no config file for : redis
[2017-07-07 16:31:54,870][WARN ][main]no config file for : ftp
[2017-07-07 16:32:39,816][WARN ][main]no config file for : redis
[2017-07-07 16:32:39,873][WARN ][main]no config file for : ftp
[2017-07-07 16:33:50,720][WARN ][main]no config file for : redis
[2017-07-07 16:33:50,777][WARN ][main]no config file for : ftp
[2017-07-07 16:33:50,790][WARN ][main]no config file for : ds
[2017-07-07 16:34:36,993][WARN ][main]no config file for : redis
[2017-07-07 16:34:37,050][WARN ][main]no config file for : ftp
[2017-07-07 16:34:37,064][WARN ][main]no config file for : ds
[2017-07-07 16:39:04,037][WARN ][main]no config file for : redis
[2017-07-07 16:39:04,090][WARN ][main]no config file for : ftp
[2017-07-07 16:39:04,103][WARN ][main]no config file for : ds
[2017-07-07 16:39:17,446][WARN ][main]no config file for : redis
[2017-07-07 16:39:17,503][WARN ][main]no config file for : ftp
[2017-07-07 16:39:17,516][WARN ][main]no config file for : ds
[2017-07-07 16:40:01,135][WARN ][main]no config file for : redis
[2017-07-07 16:40:01,192][WARN ][main]no config file for : ftp
[2017-07-07 16:40:01,206][WARN ][main]no config file for : ds
[2017-07-07 16:44:04,864][WARN ][main]no config file for : redis
[2017-07-07 16:44:04,921][WARN ][main]no config file for : ftp
[2017-07-07 16:44:04,934][WARN ][main]no config file for : ds
[2017-07-07 16:44:48,094][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,094][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,094][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,094][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,094][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,094][INFO ][main]com.mysql.jdbc.Util.getInstance(Util.java:381)
[2017-07-07 16:44:48,094][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1030)
[2017-07-07 16:44:48,094][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2017-07-07 16:44:48,094][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
[2017-07-07 16:44:48,094][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
[2017-07-07 16:44:48,095][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
[2017-07-07 16:44:48,095][INFO ][main]com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
[2017-07-07 16:44:48,095][INFO ][main]com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
[2017-07-07 16:44:48,095][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
[2017-07-07 16:44:48,095][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-07-07 16:44:48,095][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-07-07 16:44:48,095][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,095][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,095][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,095][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,095][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,095][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-07-07 16:44:48,095][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-07-07 16:44:48,096][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:571)
[2017-07-07 16:44:48,096][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:187)
[2017-07-07 16:44:48,096][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-07-07 16:44:48,096][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-07-07 16:44:48,096][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-07-07 16:44:48,096][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-07-07 16:44:48,096][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-07-07 16:44:48,096][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-07-07 16:44:48,096][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-07-07 16:44:48,096][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-07-07 16:44:48,096][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-07-07 16:44:48,096][INFO ][main]Test.main(Test.java:18)
[2017-07-07 16:44:48,096][ERROR][main]com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Access denied for user 'cfcrm_user'@'%' to database 'cfcrmdebug'
[2017-07-07 16:44:48,112][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,112][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,112][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,112][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,112][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,112][INFO ][main]com.mysql.jdbc.Util.getInstance(Util.java:381)
[2017-07-07 16:44:48,112][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1030)
[2017-07-07 16:44:48,112][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2017-07-07 16:44:48,112][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
[2017-07-07 16:44:48,112][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
[2017-07-07 16:44:48,112][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
[2017-07-07 16:44:48,112][INFO ][main]com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
[2017-07-07 16:44:48,113][INFO ][main]com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
[2017-07-07 16:44:48,113][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
[2017-07-07 16:44:48,113][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-07-07 16:44:48,113][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-07-07 16:44:48,113][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,113][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,113][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,113][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,113][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,113][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-07-07 16:44:48,113][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-07-07 16:44:48,113][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:571)
[2017-07-07 16:44:48,113][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:187)
[2017-07-07 16:44:48,113][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-07-07 16:44:48,114][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-07-07 16:44:48,114][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-07-07 16:44:48,114][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-07-07 16:44:48,114][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-07-07 16:44:48,114][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-07-07 16:44:48,114][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-07-07 16:44:48,114][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-07-07 16:44:48,114][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-07-07 16:44:48,114][INFO ][main]Test.main(Test.java:18)
[2017-07-07 16:44:48,114][ERROR][main]com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Access denied for user 'cfcrm_user'@'%' to database 'cfcrmdebug'
[2017-07-07 16:44:48,130][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,130][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,130][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,130][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,130][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,130][INFO ][main]com.mysql.jdbc.Util.getInstance(Util.java:381)
[2017-07-07 16:44:48,130][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1030)
[2017-07-07 16:44:48,130][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2017-07-07 16:44:48,130][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
[2017-07-07 16:44:48,130][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
[2017-07-07 16:44:48,131][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
[2017-07-07 16:44:48,131][INFO ][main]com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
[2017-07-07 16:44:48,131][INFO ][main]com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
[2017-07-07 16:44:48,131][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
[2017-07-07 16:44:48,131][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-07-07 16:44:48,131][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-07-07 16:44:48,131][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,131][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,131][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,131][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,131][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,131][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-07-07 16:44:48,131][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-07-07 16:44:48,131][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:571)
[2017-07-07 16:44:48,131][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:187)
[2017-07-07 16:44:48,131][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-07-07 16:44:48,131][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-07-07 16:44:48,131][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-07-07 16:44:48,132][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-07-07 16:44:48,132][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-07-07 16:44:48,132][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-07-07 16:44:48,132][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-07-07 16:44:48,132][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-07-07 16:44:48,132][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-07-07 16:44:48,132][INFO ][main]Test.main(Test.java:18)
[2017-07-07 16:44:48,132][ERROR][main]com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Access denied for user 'cfcrm_user'@'%' to database 'cfcrmdebug'
[2017-07-07 16:44:48,149][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,149][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,149][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,149][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,149][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,149][INFO ][main]com.mysql.jdbc.Util.getInstance(Util.java:381)
[2017-07-07 16:44:48,149][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1030)
[2017-07-07 16:44:48,149][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2017-07-07 16:44:48,150][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
[2017-07-07 16:44:48,150][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
[2017-07-07 16:44:48,150][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
[2017-07-07 16:44:48,150][INFO ][main]com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
[2017-07-07 16:44:48,150][INFO ][main]com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
[2017-07-07 16:44:48,150][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
[2017-07-07 16:44:48,150][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-07-07 16:44:48,150][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-07-07 16:44:48,150][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,150][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,150][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,150][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,150][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,150][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-07-07 16:44:48,151][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-07-07 16:44:48,151][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:571)
[2017-07-07 16:44:48,151][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:187)
[2017-07-07 16:44:48,151][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-07-07 16:44:48,151][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-07-07 16:44:48,151][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-07-07 16:44:48,151][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-07-07 16:44:48,151][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-07-07 16:44:48,151][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-07-07 16:44:48,151][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-07-07 16:44:48,151][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-07-07 16:44:48,151][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-07-07 16:44:48,151][INFO ][main]Test.main(Test.java:18)
[2017-07-07 16:44:48,151][ERROR][main]com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Access denied for user 'cfcrm_user'@'%' to database 'cfcrmdebug'
[2017-07-07 16:44:48,166][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,166][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,166][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,166][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,166][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,166][INFO ][main]com.mysql.jdbc.Util.getInstance(Util.java:381)
[2017-07-07 16:44:48,166][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1030)
[2017-07-07 16:44:48,166][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2017-07-07 16:44:48,166][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
[2017-07-07 16:44:48,166][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
[2017-07-07 16:44:48,166][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
[2017-07-07 16:44:48,166][INFO ][main]com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
[2017-07-07 16:44:48,166][INFO ][main]com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
[2017-07-07 16:44:48,166][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
[2017-07-07 16:44:48,166][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-07-07 16:44:48,166][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-07-07 16:44:48,167][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,167][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,167][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,167][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,167][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,167][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-07-07 16:44:48,167][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-07-07 16:44:48,167][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:571)
[2017-07-07 16:44:48,167][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:187)
[2017-07-07 16:44:48,167][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-07-07 16:44:48,167][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-07-07 16:44:48,167][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-07-07 16:44:48,167][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-07-07 16:44:48,167][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-07-07 16:44:48,167][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-07-07 16:44:48,167][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-07-07 16:44:48,167][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-07-07 16:44:48,167][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-07-07 16:44:48,167][INFO ][main]Test.main(Test.java:18)
[2017-07-07 16:44:48,167][ERROR][main]com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Access denied for user 'cfcrm_user'@'%' to database 'cfcrmdebug'
[2017-07-07 16:44:48,187][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,187][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,187][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,187][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,188][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,188][INFO ][main]com.mysql.jdbc.Util.getInstance(Util.java:381)
[2017-07-07 16:44:48,188][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1030)
[2017-07-07 16:44:48,188][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2017-07-07 16:44:48,188][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
[2017-07-07 16:44:48,188][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
[2017-07-07 16:44:48,188][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
[2017-07-07 16:44:48,188][INFO ][main]com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
[2017-07-07 16:44:48,188][INFO ][main]com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
[2017-07-07 16:44:48,188][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
[2017-07-07 16:44:48,188][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-07-07 16:44:48,188][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-07-07 16:44:48,188][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,188][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,188][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,188][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,188][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,189][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-07-07 16:44:48,189][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-07-07 16:44:48,189][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:571)
[2017-07-07 16:44:48,189][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:187)
[2017-07-07 16:44:48,189][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-07-07 16:44:48,189][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-07-07 16:44:48,189][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-07-07 16:44:48,189][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-07-07 16:44:48,189][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-07-07 16:44:48,189][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-07-07 16:44:48,189][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-07-07 16:44:48,189][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-07-07 16:44:48,189][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-07-07 16:44:48,189][INFO ][main]Test.main(Test.java:18)
[2017-07-07 16:44:48,189][ERROR][main]com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Access denied for user 'cfcrm_user'@'%' to database 'cfcrmdebug'
[2017-07-07 16:44:48,205][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,205][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,205][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,205][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,205][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,205][INFO ][main]com.mysql.jdbc.Util.getInstance(Util.java:381)
[2017-07-07 16:44:48,205][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1030)
[2017-07-07 16:44:48,205][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2017-07-07 16:44:48,205][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
[2017-07-07 16:44:48,205][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
[2017-07-07 16:44:48,206][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
[2017-07-07 16:44:48,206][INFO ][main]com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
[2017-07-07 16:44:48,206][INFO ][main]com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
[2017-07-07 16:44:48,206][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
[2017-07-07 16:44:48,206][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-07-07 16:44:48,206][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-07-07 16:44:48,206][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,206][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,206][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,206][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,206][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,206][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-07-07 16:44:48,206][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-07-07 16:44:48,206][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:571)
[2017-07-07 16:44:48,206][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:187)
[2017-07-07 16:44:48,206][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-07-07 16:44:48,206][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-07-07 16:44:48,207][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-07-07 16:44:48,207][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-07-07 16:44:48,207][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-07-07 16:44:48,207][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-07-07 16:44:48,207][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-07-07 16:44:48,207][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-07-07 16:44:48,207][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-07-07 16:44:48,207][INFO ][main]Test.main(Test.java:18)
[2017-07-07 16:44:48,207][ERROR][main]com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Access denied for user 'cfcrm_user'@'%' to database 'cfcrmdebug'
[2017-07-07 16:44:48,222][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,222][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,222][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,222][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,222][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,222][INFO ][main]com.mysql.jdbc.Util.getInstance(Util.java:381)
[2017-07-07 16:44:48,222][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1030)
[2017-07-07 16:44:48,222][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2017-07-07 16:44:48,222][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
[2017-07-07 16:44:48,222][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
[2017-07-07 16:44:48,222][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
[2017-07-07 16:44:48,222][INFO ][main]com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
[2017-07-07 16:44:48,222][INFO ][main]com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
[2017-07-07 16:44:48,222][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
[2017-07-07 16:44:48,222][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-07-07 16:44:48,223][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-07-07 16:44:48,223][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,223][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,223][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,223][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,223][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,223][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-07-07 16:44:48,223][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-07-07 16:44:48,223][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:571)
[2017-07-07 16:44:48,223][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:187)
[2017-07-07 16:44:48,223][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-07-07 16:44:48,224][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-07-07 16:44:48,224][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-07-07 16:44:48,224][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-07-07 16:44:48,224][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-07-07 16:44:48,224][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-07-07 16:44:48,224][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-07-07 16:44:48,224][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-07-07 16:44:48,224][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-07-07 16:44:48,224][INFO ][main]Test.main(Test.java:18)
[2017-07-07 16:44:48,224][ERROR][main]com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Access denied for user 'cfcrm_user'@'%' to database 'cfcrmdebug'
[2017-07-07 16:44:48,239][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,239][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,239][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,239][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,239][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,239][INFO ][main]com.mysql.jdbc.Util.getInstance(Util.java:381)
[2017-07-07 16:44:48,239][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1030)
[2017-07-07 16:44:48,239][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2017-07-07 16:44:48,240][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
[2017-07-07 16:44:48,240][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
[2017-07-07 16:44:48,240][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
[2017-07-07 16:44:48,240][INFO ][main]com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
[2017-07-07 16:44:48,240][INFO ][main]com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
[2017-07-07 16:44:48,240][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
[2017-07-07 16:44:48,240][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-07-07 16:44:48,240][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-07-07 16:44:48,240][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,240][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,240][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,240][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,240][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,241][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-07-07 16:44:48,241][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-07-07 16:44:48,241][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:571)
[2017-07-07 16:44:48,241][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:187)
[2017-07-07 16:44:48,241][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-07-07 16:44:48,241][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-07-07 16:44:48,241][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-07-07 16:44:48,241][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-07-07 16:44:48,241][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-07-07 16:44:48,241][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-07-07 16:44:48,241][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-07-07 16:44:48,241][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-07-07 16:44:48,241][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-07-07 16:44:48,241][INFO ][main]Test.main(Test.java:18)
[2017-07-07 16:44:48,242][ERROR][main]com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Access denied for user 'cfcrm_user'@'%' to database 'cfcrmdebug'
[2017-07-07 16:44:48,257][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,257][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,257][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,257][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,257][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,257][INFO ][main]com.mysql.jdbc.Util.getInstance(Util.java:381)
[2017-07-07 16:44:48,257][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1030)
[2017-07-07 16:44:48,257][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2017-07-07 16:44:48,257][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
[2017-07-07 16:44:48,257][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
[2017-07-07 16:44:48,257][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
[2017-07-07 16:44:48,257][INFO ][main]com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
[2017-07-07 16:44:48,257][INFO ][main]com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
[2017-07-07 16:44:48,257][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
[2017-07-07 16:44:48,257][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-07-07 16:44:48,257][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-07-07 16:44:48,257][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,258][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,258][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,258][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,258][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,258][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-07-07 16:44:48,258][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-07-07 16:44:48,258][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:571)
[2017-07-07 16:44:48,258][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:187)
[2017-07-07 16:44:48,258][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-07-07 16:44:48,258][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-07-07 16:44:48,258][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-07-07 16:44:48,258][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-07-07 16:44:48,258][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-07-07 16:44:48,258][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-07-07 16:44:48,258][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-07-07 16:44:48,258][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-07-07 16:44:48,258][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-07-07 16:44:48,258][INFO ][main]Test.main(Test.java:18)
[2017-07-07 16:44:48,258][ERROR][main]com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Access denied for user 'cfcrm_user'@'%' to database 'cfcrmdebug'
[2017-07-07 16:44:48,276][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,276][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,276][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,276][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,276][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,276][INFO ][main]com.mysql.jdbc.Util.getInstance(Util.java:381)
[2017-07-07 16:44:48,276][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1030)
[2017-07-07 16:44:48,276][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2017-07-07 16:44:48,276][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
[2017-07-07 16:44:48,276][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
[2017-07-07 16:44:48,276][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
[2017-07-07 16:44:48,277][INFO ][main]com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
[2017-07-07 16:44:48,277][INFO ][main]com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
[2017-07-07 16:44:48,277][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
[2017-07-07 16:44:48,277][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-07-07 16:44:48,277][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-07-07 16:44:48,277][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,277][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,277][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,277][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,277][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,277][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-07-07 16:44:48,277][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-07-07 16:44:48,277][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:571)
[2017-07-07 16:44:48,277][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:187)
[2017-07-07 16:44:48,277][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-07-07 16:44:48,277][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-07-07 16:44:48,277][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-07-07 16:44:48,277][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-07-07 16:44:48,277][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-07-07 16:44:48,277][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-07-07 16:44:48,278][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-07-07 16:44:48,278][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-07-07 16:44:48,278][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-07-07 16:44:48,278][INFO ][main]Test.main(Test.java:18)
[2017-07-07 16:44:48,278][ERROR][main]com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Access denied for user 'cfcrm_user'@'%' to database 'cfcrmdebug'
[2017-07-07 16:44:48,291][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,291][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,291][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,291][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,292][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,292][INFO ][main]com.mysql.jdbc.Util.getInstance(Util.java:381)
[2017-07-07 16:44:48,292][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1030)
[2017-07-07 16:44:48,292][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2017-07-07 16:44:48,292][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
[2017-07-07 16:44:48,292][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
[2017-07-07 16:44:48,292][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
[2017-07-07 16:44:48,292][INFO ][main]com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
[2017-07-07 16:44:48,292][INFO ][main]com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
[2017-07-07 16:44:48,292][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
[2017-07-07 16:44:48,292][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-07-07 16:44:48,292][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-07-07 16:44:48,292][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,292][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,292][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,292][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,292][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,292][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-07-07 16:44:48,292][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-07-07 16:44:48,292][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:571)
[2017-07-07 16:44:48,292][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:187)
[2017-07-07 16:44:48,293][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-07-07 16:44:48,293][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-07-07 16:44:48,293][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-07-07 16:44:48,293][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-07-07 16:44:48,293][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-07-07 16:44:48,293][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-07-07 16:44:48,293][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-07-07 16:44:48,293][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-07-07 16:44:48,293][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-07-07 16:44:48,293][INFO ][main]Test.main(Test.java:18)
[2017-07-07 16:44:48,293][ERROR][main]com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Access denied for user 'cfcrm_user'@'%' to database 'cfcrmdebug'
[2017-07-07 16:44:48,309][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,309][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,309][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,309][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,309][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,309][INFO ][main]com.mysql.jdbc.Util.getInstance(Util.java:381)
[2017-07-07 16:44:48,309][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1030)
[2017-07-07 16:44:48,309][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2017-07-07 16:44:48,309][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
[2017-07-07 16:44:48,309][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
[2017-07-07 16:44:48,309][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
[2017-07-07 16:44:48,309][INFO ][main]com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
[2017-07-07 16:44:48,309][INFO ][main]com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
[2017-07-07 16:44:48,309][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
[2017-07-07 16:44:48,309][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-07-07 16:44:48,310][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-07-07 16:44:48,310][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,310][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,310][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,310][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,310][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,310][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-07-07 16:44:48,310][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-07-07 16:44:48,310][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:571)
[2017-07-07 16:44:48,310][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:187)
[2017-07-07 16:44:48,310][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-07-07 16:44:48,310][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-07-07 16:44:48,310][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-07-07 16:44:48,310][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-07-07 16:44:48,310][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-07-07 16:44:48,310][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-07-07 16:44:48,310][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-07-07 16:44:48,310][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-07-07 16:44:48,310][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-07-07 16:44:48,310][INFO ][main]Test.main(Test.java:18)
[2017-07-07 16:44:48,310][ERROR][main]com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Access denied for user 'cfcrm_user'@'%' to database 'cfcrmdebug'
[2017-07-07 16:44:48,325][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,325][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,325][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,325][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,325][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,325][INFO ][main]com.mysql.jdbc.Util.getInstance(Util.java:381)
[2017-07-07 16:44:48,325][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1030)
[2017-07-07 16:44:48,325][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2017-07-07 16:44:48,325][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
[2017-07-07 16:44:48,326][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
[2017-07-07 16:44:48,326][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
[2017-07-07 16:44:48,326][INFO ][main]com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
[2017-07-07 16:44:48,326][INFO ][main]com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
[2017-07-07 16:44:48,326][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
[2017-07-07 16:44:48,326][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-07-07 16:44:48,326][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-07-07 16:44:48,326][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,326][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,326][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,326][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,326][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,326][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-07-07 16:44:48,326][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-07-07 16:44:48,326][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:571)
[2017-07-07 16:44:48,327][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:187)
[2017-07-07 16:44:48,327][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-07-07 16:44:48,327][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-07-07 16:44:48,327][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-07-07 16:44:48,327][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-07-07 16:44:48,327][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-07-07 16:44:48,327][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-07-07 16:44:48,327][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-07-07 16:44:48,327][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-07-07 16:44:48,327][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-07-07 16:44:48,327][INFO ][main]Test.main(Test.java:18)
[2017-07-07 16:44:48,327][ERROR][main]com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Access denied for user 'cfcrm_user'@'%' to database 'cfcrmdebug'
[2017-07-07 16:44:48,398][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,398][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,398][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,398][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,398][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,398][INFO ][main]com.mysql.jdbc.Util.getInstance(Util.java:381)
[2017-07-07 16:44:48,399][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1030)
[2017-07-07 16:44:48,399][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2017-07-07 16:44:48,399][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
[2017-07-07 16:44:48,399][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
[2017-07-07 16:44:48,399][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
[2017-07-07 16:44:48,399][INFO ][main]com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
[2017-07-07 16:44:48,399][INFO ][main]com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
[2017-07-07 16:44:48,399][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
[2017-07-07 16:44:48,399][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-07-07 16:44:48,399][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-07-07 16:44:48,399][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,399][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,399][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,399][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,399][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,399][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-07-07 16:44:48,399][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-07-07 16:44:48,399][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:571)
[2017-07-07 16:44:48,399][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:187)
[2017-07-07 16:44:48,399][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-07-07 16:44:48,399][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-07-07 16:44:48,399][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-07-07 16:44:48,400][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-07-07 16:44:48,400][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-07-07 16:44:48,400][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-07-07 16:44:48,400][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-07-07 16:44:48,400][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-07-07 16:44:48,400][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-07-07 16:44:48,400][INFO ][main]Test.main(Test.java:18)
[2017-07-07 16:44:48,400][ERROR][main]com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Access denied for user 'cfcrm_user'@'%' to database 'cfcrmdebug'
[2017-07-07 16:44:48,414][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2017-07-07 16:44:48,414][INFO ][main]sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
[2017-07-07 16:44:48,414][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,414][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,414][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,414][INFO ][main]com.mysql.jdbc.Util.getInstance(Util.java:381)
[2017-07-07 16:44:48,414][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1030)
[2017-07-07 16:44:48,414][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2017-07-07 16:44:48,415][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
[2017-07-07 16:44:48,415][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
[2017-07-07 16:44:48,415][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
[2017-07-07 16:44:48,415][INFO ][main]com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
[2017-07-07 16:44:48,415][INFO ][main]com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
[2017-07-07 16:44:48,415][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
[2017-07-07 16:44:48,415][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-07-07 16:44:48,415][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-07-07 16:44:48,415][INFO ][main]sun.reflect.GeneratedConstructorAccessor55.newInstance(Unknown Source)
[2017-07-07 16:44:48,415][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,415][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,415][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,415][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-07-07 16:44:48,415][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-07-07 16:44:48,415][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:571)
[2017-07-07 16:44:48,415][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:187)
[2017-07-07 16:44:48,415][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-07-07 16:44:48,415][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-07-07 16:44:48,415][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-07-07 16:44:48,415][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-07-07 16:44:48,415][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-07-07 16:44:48,416][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-07-07 16:44:48,416][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-07-07 16:44:48,416][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-07-07 16:44:48,416][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-07-07 16:44:48,416][INFO ][main]Test.main(Test.java:18)
[2017-07-07 16:44:48,416][ERROR][main]com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Access denied for user 'cfcrm_user'@'%' to database 'cfcrmdebug'
[2017-07-07 16:44:48,432][INFO ][main]sun.reflect.GeneratedConstructorAccessor57.newInstance(Unknown Source)
[2017-07-07 16:44:48,432][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,432][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,432][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,432][INFO ][main]com.mysql.jdbc.Util.getInstance(Util.java:381)
[2017-07-07 16:44:48,432][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1030)
[2017-07-07 16:44:48,432][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2017-07-07 16:44:48,432][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
[2017-07-07 16:44:48,432][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
[2017-07-07 16:44:48,432][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
[2017-07-07 16:44:48,432][INFO ][main]com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
[2017-07-07 16:44:48,432][INFO ][main]com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
[2017-07-07 16:44:48,432][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
[2017-07-07 16:44:48,432][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-07-07 16:44:48,432][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-07-07 16:44:48,432][INFO ][main]sun.reflect.GeneratedConstructorAccessor55.newInstance(Unknown Source)
[2017-07-07 16:44:48,432][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,432][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,433][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,433][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-07-07 16:44:48,433][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-07-07 16:44:48,433][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:571)
[2017-07-07 16:44:48,433][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:187)
[2017-07-07 16:44:48,433][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-07-07 16:44:48,433][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-07-07 16:44:48,433][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-07-07 16:44:48,433][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-07-07 16:44:48,433][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-07-07 16:44:48,433][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-07-07 16:44:48,433][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-07-07 16:44:48,433][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-07-07 16:44:48,433][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-07-07 16:44:48,433][INFO ][main]Test.main(Test.java:18)
[2017-07-07 16:44:48,433][ERROR][main]com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Access denied for user 'cfcrm_user'@'%' to database 'cfcrmdebug'
[2017-07-07 16:44:48,446][INFO ][main]sun.reflect.GeneratedConstructorAccessor57.newInstance(Unknown Source)
[2017-07-07 16:44:48,447][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,447][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,447][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,447][INFO ][main]com.mysql.jdbc.Util.getInstance(Util.java:381)
[2017-07-07 16:44:48,447][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1030)
[2017-07-07 16:44:48,447][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2017-07-07 16:44:48,447][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
[2017-07-07 16:44:48,447][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
[2017-07-07 16:44:48,447][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
[2017-07-07 16:44:48,447][INFO ][main]com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
[2017-07-07 16:44:48,447][INFO ][main]com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
[2017-07-07 16:44:48,447][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
[2017-07-07 16:44:48,447][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-07-07 16:44:48,447][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-07-07 16:44:48,447][INFO ][main]sun.reflect.GeneratedConstructorAccessor55.newInstance(Unknown Source)
[2017-07-07 16:44:48,447][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,447][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,447][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,447][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-07-07 16:44:48,448][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-07-07 16:44:48,448][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:571)
[2017-07-07 16:44:48,448][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:187)
[2017-07-07 16:44:48,448][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-07-07 16:44:48,448][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-07-07 16:44:48,448][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-07-07 16:44:48,448][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-07-07 16:44:48,448][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-07-07 16:44:48,448][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-07-07 16:44:48,448][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-07-07 16:44:48,448][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-07-07 16:44:48,448][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-07-07 16:44:48,448][INFO ][main]Test.main(Test.java:18)
[2017-07-07 16:44:48,448][ERROR][main]com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Access denied for user 'cfcrm_user'@'%' to database 'cfcrmdebug'
[2017-07-07 16:44:48,464][INFO ][main]sun.reflect.GeneratedConstructorAccessor57.newInstance(Unknown Source)
[2017-07-07 16:44:48,465][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,465][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,465][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,465][INFO ][main]com.mysql.jdbc.Util.getInstance(Util.java:381)
[2017-07-07 16:44:48,465][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1030)
[2017-07-07 16:44:48,465][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2017-07-07 16:44:48,465][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
[2017-07-07 16:44:48,465][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
[2017-07-07 16:44:48,465][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
[2017-07-07 16:44:48,465][INFO ][main]com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
[2017-07-07 16:44:48,465][INFO ][main]com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
[2017-07-07 16:44:48,466][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
[2017-07-07 16:44:48,466][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-07-07 16:44:48,466][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-07-07 16:44:48,466][INFO ][main]sun.reflect.GeneratedConstructorAccessor55.newInstance(Unknown Source)
[2017-07-07 16:44:48,466][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,466][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,466][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,466][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-07-07 16:44:48,466][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-07-07 16:44:48,466][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:571)
[2017-07-07 16:44:48,466][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:187)
[2017-07-07 16:44:48,466][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-07-07 16:44:48,466][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-07-07 16:44:48,466][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-07-07 16:44:48,466][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-07-07 16:44:48,466][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-07-07 16:44:48,466][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-07-07 16:44:48,466][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-07-07 16:44:48,466][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-07-07 16:44:48,466][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-07-07 16:44:48,466][INFO ][main]Test.main(Test.java:18)
[2017-07-07 16:44:48,466][ERROR][main]com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Access denied for user 'cfcrm_user'@'%' to database 'cfcrmdebug'
[2017-07-07 16:44:48,483][INFO ][main]sun.reflect.GeneratedConstructorAccessor57.newInstance(Unknown Source)
[2017-07-07 16:44:48,483][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,483][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,483][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,483][INFO ][main]com.mysql.jdbc.Util.getInstance(Util.java:381)
[2017-07-07 16:44:48,483][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1030)
[2017-07-07 16:44:48,483][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2017-07-07 16:44:48,483][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
[2017-07-07 16:44:48,483][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
[2017-07-07 16:44:48,483][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
[2017-07-07 16:44:48,483][INFO ][main]com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
[2017-07-07 16:44:48,483][INFO ][main]com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
[2017-07-07 16:44:48,484][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
[2017-07-07 16:44:48,484][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-07-07 16:44:48,484][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-07-07 16:44:48,484][INFO ][main]sun.reflect.GeneratedConstructorAccessor55.newInstance(Unknown Source)
[2017-07-07 16:44:48,484][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,484][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,484][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,484][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-07-07 16:44:48,484][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-07-07 16:44:48,484][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:571)
[2017-07-07 16:44:48,484][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:187)
[2017-07-07 16:44:48,484][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-07-07 16:44:48,484][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-07-07 16:44:48,484][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-07-07 16:44:48,484][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-07-07 16:44:48,484][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-07-07 16:44:48,484][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-07-07 16:44:48,484][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-07-07 16:44:48,484][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-07-07 16:44:48,484][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-07-07 16:44:48,485][INFO ][main]Test.main(Test.java:18)
[2017-07-07 16:44:48,485][ERROR][main]com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Access denied for user 'cfcrm_user'@'%' to database 'cfcrmdebug'
[2017-07-07 16:44:48,498][INFO ][main]sun.reflect.GeneratedConstructorAccessor57.newInstance(Unknown Source)
[2017-07-07 16:44:48,499][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,499][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,499][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,499][INFO ][main]com.mysql.jdbc.Util.getInstance(Util.java:381)
[2017-07-07 16:44:48,499][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1030)
[2017-07-07 16:44:48,499][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2017-07-07 16:44:48,499][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
[2017-07-07 16:44:48,499][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
[2017-07-07 16:44:48,499][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
[2017-07-07 16:44:48,499][INFO ][main]com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
[2017-07-07 16:44:48,499][INFO ][main]com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
[2017-07-07 16:44:48,499][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
[2017-07-07 16:44:48,499][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-07-07 16:44:48,499][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-07-07 16:44:48,499][INFO ][main]sun.reflect.GeneratedConstructorAccessor55.newInstance(Unknown Source)
[2017-07-07 16:44:48,500][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:48,500][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:48,500][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:48,500][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-07-07 16:44:48,500][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-07-07 16:44:48,500][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:571)
[2017-07-07 16:44:48,500][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:187)
[2017-07-07 16:44:48,500][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-07-07 16:44:48,500][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-07-07 16:44:48,500][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-07-07 16:44:48,500][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTable(MysqlTableSrcUtil.java:61)
[2017-07-07 16:44:48,500][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTable(TableSrcUtilFactory.java:33)
[2017-07-07 16:44:48,500][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.excutor.MysqlDBExcutor.getSql(MysqlDBExcutor.java:91)
[2017-07-07 16:44:48,500][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getSql(DBUtil.java:62)
[2017-07-07 16:44:48,500][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:43)
[2017-07-07 16:44:48,500][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-07-07 16:44:48,500][INFO ][main]Test.main(Test.java:18)
[2017-07-07 16:44:48,500][ERROR][main]com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Access denied for user 'cfcrm_user'@'%' to database 'cfcrmdebug'
[2017-07-07 16:44:53,718][INFO ][main]sun.reflect.GeneratedConstructorAccessor57.newInstance(Unknown Source)
[2017-07-07 16:44:53,718][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:53,719][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:53,719][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:53,719][INFO ][main]com.mysql.jdbc.Util.getInstance(Util.java:381)
[2017-07-07 16:44:53,719][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1030)
[2017-07-07 16:44:53,719][INFO ][main]com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
[2017-07-07 16:44:53,719][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
[2017-07-07 16:44:53,719][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
[2017-07-07 16:44:53,719][INFO ][main]com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
[2017-07-07 16:44:53,719][INFO ][main]com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
[2017-07-07 16:44:53,719][INFO ][main]com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
[2017-07-07 16:44:53,719][INFO ][main]com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
[2017-07-07 16:44:53,719][INFO ][main]com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
[2017-07-07 16:44:53,719][INFO ][main]com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
[2017-07-07 16:44:53,719][INFO ][main]sun.reflect.GeneratedConstructorAccessor55.newInstance(Unknown Source)
[2017-07-07 16:44:53,719][INFO ][main]sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2017-07-07 16:44:53,719][INFO ][main]java.lang.reflect.Constructor.newInstance(Constructor.java:526)
[2017-07-07 16:44:53,719][INFO ][main]com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[2017-07-07 16:44:53,719][INFO ][main]com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
[2017-07-07 16:44:53,719][INFO ][main]com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
[2017-07-07 16:44:53,719][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:571)
[2017-07-07 16:44:53,719][INFO ][main]java.sql.DriverManager.getConnection(DriverManager.java:187)
[2017-07-07 16:44:53,719][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createConnection(DBSrc.java:119)
[2017-07-07 16:44:53,720][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.createMyConnection(DBSrc.java:106)
[2017-07-07 16:44:53,720][INFO ][main]com.tuhanbao.base.util.db.conn.DBSrc.getConnection(DBSrc.java:171)
[2017-07-07 16:44:53,720][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.MysqlTableSrcUtil.getTables(MysqlTableSrcUtil.java:41)
[2017-07-07 16:44:53,720][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.tableUtil.src.TableSrcUtilFactory.getTables(TableSrcUtilFactory.java:44)
[2017-07-07 16:44:53,720][INFO ][main]com.tuhanbao.base.util.io.codeGenarator.sqlUtil.DBUtil.getTables(DBUtil.java:152)
[2017-07-07 16:44:53,720][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:48)
[2017-07-07 16:44:53,720][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-07-07 16:44:53,720][INFO ][main]Test.main(Test.java:18)
[2017-07-07 16:44:53,720][ERROR][main]com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Access denied for user 'cfcrm_user'@'%' to database 'cfcrmdebug'
[2017-07-07 16:44:53,720][INFO ][main]com.tuhanbao.autotool.mvc.excel.SqlGeneratorFilter.filter(SqlGeneratorFilter.java:50)
[2017-07-07 16:44:53,720][INFO ][main]com.tuhanbao.base.chain.Context.start(Context.java:61)
[2017-07-07 16:44:53,720][INFO ][main]Test.main(Test.java:18)
[2017-07-07 16:44:53,720][ERROR][main]java.lang.NullPointerException
[2017-07-07 16:46:29,417][WARN ][main]no config file for : redis
[2017-07-07 16:46:29,470][WARN ][main]no config file for : ftp
[2017-07-07 16:46:29,482][WARN ][main]no config file for : ds
[2017-07-07 17:15:56,093][WARN ][main]no config file for : redis
[2017-07-07 17:15:56,149][WARN ][main]no config file for : ftp
[2017-07-07 17:15:56,163][WARN ][main]no config file for : ds
[2017-07-07 17:17:14,809][WARN ][main]no config file for : redis
[2017-07-07 17:17:14,865][WARN ][main]no config file for : ftp
[2017-07-07 17:17:14,879][WARN ][main]no config file for : ds
[2017-07-07 17:18:40,735][WARN ][main]no config file for : redis
[2017-07-07 17:18:40,793][WARN ][main]no config file for : ftp
[2017-07-07 17:18:40,805][WARN ][main]no config file for : ds
[2017-07-07 17:19:51,941][WARN ][main]no config file for : redis
[2017-07-07 17:19:51,996][WARN ][main]no config file for : ftp
[2017-07-07 17:19:52,009][WARN ][main]no config file for : ds
